mitosheet/src/utils/actions.tsx

```diff
@@ -1281,11 +1281,34 @@ export const createActions = (
             searchTerms: ['CodeSnippets'],
             tooltip: "CodeSnippets"
         },
+        [ActionEnum.RESET_AND_KEEP_INDEX]: {
+            type: ActionEnum.RESET_AND_KEEP_INDEX,
+            shortTitle: 'Reset Index',
+            longTitle: 'Reset Index',
+            actionFunction: () => {
+                void mitoAPI.editResetIndex(sheetIndex, false);
+            },
+            isDisabled: () => {return doesAnySheetExist(sheetDataArray) ? undefined : 'Import data before resetting an index.'},
+            searchTerms: ['reset', 'index'],
+            tooltip: "Resets a dataframes index to 0-NUM_ROWS. Will keep the current index as a column in the dataframe."
+        },
+        [ActionEnum.RESET_AND_DROP_INDEX]: {
+            type: ActionEnum.RESET_AND_DROP_INDEX,
+            shortTitle: 'Reset and Drop Index',
+            longTitle: 'Reset and Drop Index',
+            actionFunction: () => {
+                void mitoAPI.editResetIndex(sheetIndex, true);
+            },
+            isDisabled: () => {return doesAnySheetExist(sheetDataArray) ? undefined : 'Import data before resetting an index.'},
+            searchTerms: ['Reset Index'],
```

Comment:
I think we want our search terms to be lowercase. And to keep it consistent, with the above should be: ['reset', 'index']